petrometer/petrometer.py

```diff
@@ -41,19 +41,51 @@
 
 class Petrometer:
     def __init__(self, args: list):
-        parser = argparse.ArgumentParser(prog='petrometer')
-        parser.add_argument("addresses", metavar='ADDRESSES', nargs='+', type=str,
-                            help="Ethereum addresses to get the gas usage of")
-        parser.add_argument("--etherscan-api-key", help="Etherscan API key", required=True, type=str)
-        parser.add_argument("-j", '--json', help="Generate result as JSON", dest='json', action='store_true')
-        parser.add_argument("-o", "--output", help="File to save the output to", required=False, type=str)
-        parser.add_argument("-i", "--incoming", help="Show incoming transaction gas usage, defaut outgoing",
-                            required=False, action='store_true')
+        parser = argparse.ArgumentParser(prog="petrometer")
+        parser.add_argument(
```

Comment:
@OleksandrUA I see formatting for some of the parsing vars has changed. Can you make them all look the same, so we have consistency?